src/models/tools/table/table-content.ts

```diff
@@ -204,101 +235,121 @@ export const TableContentModel = ToolContentModel
     doPostCreate(metadata) {
       self.metadata = metadata as TableMetadataModelType;
 
-      if (self.dataSet.attributes.length >= 2) {
-        const xAttr = self.dataSet.attributes[0];
-        const xName = xAttr.name;
-        self.dataSet.attributes.forEach((attr, i) => {
-          if (i > 0) {
-            attr.formula.synchronize(xName);
-            if (attr.formula.display) {
-              self.metadata.setRawExpression(attr.id, attr.formula.display);
-            }
-            if (attr.formula.canonical) {
-              self.metadata.setExpression(attr.id, attr.formula.canonical);
-            }
-          }
-        });
-      }
+      // TODO Move this elsewhere, after the sharedModel has been created and connected
+      // if (self.dataSet.attributes.length >= 2) {
+      //   const xAttr = self.dataSet.attributes[0];
+      //   const xName = xAttr.name;
+      //   self.dataSet.attributes.forEach((attr, i) => {
+      //     if (i > 0) {
+      //       attr.formula.synchronize(xName);
+      //       if (attr.formula.display) {
+      //         self.metadata.setRawExpression(attr.id, attr.formula.display);
+      //       }
+      //       if (attr.formula.canonical) {
+      //         self.metadata.setExpression(attr.id, attr.formula.canonical);
+      //       }
+      //     }
+      //   });
+      // }
```

Comment:
Did this get moved? I think it could be moved into `updateAfterSharedModelChanges`

Reply:
I moved this into `updateAfterSharedModelChanges`, but it doesn't seem to be doing anything, so I'm keeping it commented with a note to rethink it.